packages/account/src/Components/currency-selector/radio-button.tsx

```diff
@@ -1,12 +1,8 @@
-import React, { InputHTMLAttributes, AllHTMLAttributes, ReactElement } from 'react';
+import React, { InputHTMLAttributes, AllHTMLAttributes } from 'react';
```

Comment:
Should we also avoid type destructuring?
Let’s use them as React.AllPossibleHTMLTypes :)